integration/client/src/main/java/com/cerner/beadledom/integration/client/BeadledomIntegrationClientModule.java

```diff
@@ -0,0 +1,49 @@
+package com.cerner.beadledom.integration.client;
+
+import com.cerner.beadledom.client.BeadledomClient;
+import com.cerner.beadledom.client.BeadledomClientConfiguration;
+import com.cerner.beadledom.client.BeadledomClientModule;
+import com.cerner.beadledom.client.BeadledomWebTarget;
+import com.cerner.beadledom.client.jackson.ObjectMapperClientFeatureModule;
+import com.cerner.beadledom.integration.api.HelloWorldResource;
+import com.google.inject.PrivateModule;
+import com.google.inject.Provides;
+import javax.inject.Singleton;
+
+/**
+ * An example Client Module.
+ *
+ * @author Nick Behrens
+ */
+public class BeadledomIntegrationClientModule extends PrivateModule {
```

Comment:
This should probably be changed to follow our updated guidance. There could be an `IntegrationClient` class that has a method `#helloWorld` or `#helloWorldResource` rather than directly exposing the client JAX-RS resource.

Reply:
Added a `BeadledomIntegrationClient` class: https://github.com/cerner/beadledom/pull/149/commits/0bb084b91c13cd5b595223d33244b4bb9b9d1421